tests/utilities/test_xla_device_utils.py

```diff
@@ -16,40 +16,32 @@
 import pytest
 
 import pytorch_lightning.utilities.xla_device_utils as xla_utils
-from pytorch_lightning.utilities import XLA_AVAILABLE
+from pytorch_lightning.utilities import XLA_AVAILABLE, TPU_AVAILABLE
 from tests.base.develop_utils import pl_multi_process_test
 
 if XLA_AVAILABLE:
     import torch_xla.core.xla_model as xm
 
 
-@pytest.mark.skipif(XLA_AVAILABLE, reason="test requires torch_xla to be absent")
+@pytest.mark.skipif(TPU_AVAILABLE, reason="test requires torch_xla to be absent")
```

Comment:
```suggestion
# lets hope that in or env we have installed XLA only for TPU devices, otherwise, it is testing in the cycle "if I am true test that I am true :D"
@pytest.mark.skipif(XLA_AVAILABLE, reason="test requires torch_xla to be absent")
```